src/components/login/Agreement.js

```diff
@@ -6,12 +6,56 @@ import Header from "../global/Header";
 const Agreement = () => {
   const navigate = useNavigate();
 
-  const [isServiceOn, setIsServiceOn] = useState(false);
-  const [isPersonalOn, setIsPersonalOn] = useState(false);
+  const { service, personal } = { ...localStorage };
+  const [isServiceOn, setIsServiceOn] = useState(!!service);
+  const [isPersonalOn, setIsPersonalOn] = useState(!!personal);
```

Comment:
```suggestion
personal !== null || personal !== undefined
```

or

```suggestion
Boolean(personal)
```


boolean 값으로 들어가는 변수는 boolean 값을 예상할 수 있도록 작성하면 좋을 것 같아요!